docs/content/preview/yugabyte-voyager/known-issues/postgresql.md

```diff
@@ -46,9 +46,9 @@ Review limitations and implement suggested workarounds to successfully migrate d
 - [XID functions is not supported](#xid-functions-is-not-supported)
 - [REFERENCING clause for triggers](#referencing-clause-for-triggers)
 - [BEFORE ROW triggers on partitioned tables](#before-row-triggers-on-partitioned-tables)
-- [Unsupported query construct: Advisory locks](#unsupported-query-construct-advisory-locks)
-- [Unsupported query construct: System columns](#unsupported-query-construct-system-columns)
-- [Unsupported query construct: XML functions](#unsupported-query-construct-xml-functions)
+- [Advisory locks is not yet implemented](#advisory-locks-is-not-yet-implemented)
```

Comment:
are not implemented, are not supported

according to earlier PR these links were linked from the code somehow?